src/Microsoft.ML.CpuMath/SseIntrinsics.cs

```diff
@@ -18,6 +18,11 @@
 using System.Runtime.Intrinsics;
 using System.Runtime.Intrinsics.X86;
 
+[assembly: InternalsVisibleTo("Microsoft.ML.CpuMath.PerformanceTests, PublicKey=0024000004800000940000000602" +
+    "00000024000052534131000400000100010015c01ae1f50e8cc09ba9eac9147cf8fd9fce2cfe9f8dce4f7301c4132ca9fb50ce8" +
+    "cbf1df4dc18dd4d210e4345c744ecb3365ed327efdbc52603faa5e21daa11234c8c4a73e51f03bf192544581ebe107adee3a349" +
+    "28e39d04e524a9ce729d5090bfd7dad9d10c722c0def9ccc08ff0a03790e48bcd1f9b6c476063e1966a1c4")]
```

Comment:
>1966a1c4 [](start = 83, length = 8)

Can you move this to a `Properties\AssemblyInfo.cs` file?  That is where this typically goes.  See [https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.Core/Properties/AssemblyInfo.cs#L7](https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.Core/Properties/AssemblyInfo.cs#L7) for an example.

Also (nit), the key string is usually on a single, long line.  It doesn't need to be readable.